dist/main/index.js

```diff
@@ -16376,8 +16384,13 @@ async function selectCompatibleVersion(versionSpec) {
     
     const versions = response.data;
     for (let i = 0; i < versions.length; i++) {
-        if (versionSpec === undefined || versionSpec === "latest"
-            || semver.satisfies(versions[i].tag_name, versionSpec)) {
+        if (versions[i].draft || versions[i].assets?.length === 0) {
```

Comment:
Note: I used a three `if` approach as it gives a much better readability compared to one unique `if` with multiple cases.